packages/api/src/command/medical/patient/consolidated-webhook.ts

```diff
@@ -69,15 +68,16 @@ export const processConsolidatedDataWebhook = async ({
       payload,
     });
     // send it to the customer and update the WH request status
-    await processRequest(
-      webhookRequest,
-      settings,
-      bundle
-        ? {
-            bundleLength: optionalToString(bundle.entry?.length ?? bundle.total) ?? "unknown",
-          }
-        : undefined
-    );
+    if (!currentPatient.data.whOverride)
```

Comment:
This prop name is not clear as to what it does. What about `disableWH`?

Reply:
Sgtm!